api/v1alpha1/hivelocitymachine_types.go

```diff
@@ -87,8 +88,11 @@ type HivelocityMachineSpec struct {
 	// +optional
 	ProviderID *string `json:"providerID,omitempty"`
 
-	// Type is the Hivelocity Machine Type for this machine.
-	Type HivelocityDeviceType `json:"type"`
+	// DeviceSelector specifies matching criteria for labels on BaremetalDevice.
```

Comment:
you don't need this extensively at all. You have the same thing below. You can shorten this comment or omit it fully